ee/vellum_cli/ping.py

```diff
@@ -14,7 +14,10 @@ def ping_command():
     try:
         workspace = client.workspaces.workspace_identity()
         organization = client.organizations.organization_identity()
-    except ApiError:
+    except ApiError as e:
+        # If user did not provide an API key, we will get a 403 error
+        if e.status_code == 403:
+            raise e
```

Comment:
[nit] can we include copy in the error message instructing the user to set their `VELLUM_API_KEY`?

We should also catch 401 here too - the fact that our django server returns a 403 here is a bug